src/js/location.js

```diff
@@ -0,0 +1,59 @@
+export class Location {
+    constructor(lat, long, language, cityName, country) {
+        this.latitude = lat;
+        this.longitude = long;
+        this.language = language;
+        this.cityName = cityName;
+        this.country = country;
+        this.timeZone = null;
+    }
+
+    static async getLocationByName(cityName, language) {
+        const response = await fetch(`https://api.mapbox.com/geocoding/v5/mapbox.places/${cityName}.json?types=place&autocomplete=false&fuzzyMatch=false&language=${language}&access_token=pk.eyJ1IjoidmFsZW50aW5hLWtyIiwiYSI6ImNrbjdzenB4bTBiaG8ycHFuNTF2ZGg5bGQifQ.bzp7eMwGHnb9ZFfDNggXCA`);
+        const jsonResponse = await response.json();
```

Comment:
could you please try to remove this await and test functionality

Reply:
it dosn't work without await